src/main/webapp/WEB-INF/views/cars/drivers/add.jsp

```diff
@@ -0,0 +1,20 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add a Driver to a Car</title>
+</head>
+<body>
+<h2>Please, fill the form below</h2>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+    DriverId<input type = "number" name = "driverId"> <br>
+    CarId<input type="number" name="carId"><br>
+    <button type="submit"> Submit </button>
+</form>
```

Comment:
Are any required fields in this form?
And maybe it is better to set the min value of ids?

Reply:
I thought for a long time how to do it, now I know how)